greenboot.spec

```diff
@@ -17,59 +17,27 @@ ExcludeArch: s390x
 BuildRequires:      systemd-rpm-macros
 %{?systemd_requires}
 Requires:           systemd >= 240
-
-%description
-%{summary}.
-
-%package auto-update-fallback
-Summary:            Automatic updates and failure fallback for rpm-ostree-based system 
-Requires:           %{name} = %{version}-%{release}
-Requires:           %{name}-reboot = %{version}-%{release}
-Requires:           %{name}-rpm-ostree-grub2 = %{version}-%{release}
-
-%description auto-update-fallback
-%{summary}.
-
-%package status
-Summary:            Message of the Day updater for greenboot
-Requires:           %{name} = %{version}-%{release}
+Requires:           grub2-tools-minimal
+Requires:           rpm-ostree
 # PAM is required to programatically read motd messages from /etc/motd.d/*
 Requires:           pam >= 1.3.1-15
 # While not strictly necessary to generate the motd, the main use-case of this package is to display it on SSH login
 Recommends:         openssh
+Obsoletes:          auto-update-fallback < 0.12.0
+Obsoletes:          grub2 < 0.12.0
```

Comment:
these need to be full package names so likely greenboot-grub2

Reply:
Also they need to have a line "Provides: greenboot-grub2" so that if someone has a script for them the base package gets pulled in instead.